### ISSUE TITLE
Include some basic info in getMetaData

### ISSUE THREAD

**Issue by AgDude** [driftyco/ionic-plugin-deploy#108]
The `getMetaData` method returns only explicitly set keys. It would be useful to include some basic information as well:

* binary version info (min, max, equivalent) for the current platform
* note
* release date

This is similar to #95, but it seems that kind of data should not need to be manually specified by the developer twice.

**Comment by ericb** [comment]
I agree that there should be a better way to gather more technical details of the deploys using the cloud client. This is all information offered via the API but it's not exactly useful in that context given you want to know this information inside the application.

I don't think it necessarily belongs with getMetadata, but there should be some way to get API level details (non-private details anyway) from the cloud client.
